### ISSUE TITLE
Possible breakage due to Discord API update

### ISSUE THREAD

**Issue by 31JSON** [nicnacnic/nodecg-discord-overlay#1]
After a recent change to the Discord API, the bot no longer responds to commands even though when running nodecg it says it's online. There is no other output or error messages. If there's something I can do to help troubleshoot, please let me know.

**Comment by nicnacnic** [comment]
Yep, I know why. The package I'm using upgraded to a new version which broke some things. In the bundle folder, try running `npm i discord.js@12.5.3` in a command line. Overall this bundle needs an update.

**Comment by 31JSON** [comment]
I haven't done a restream in a while, but I did attempt to install the newer discord.js, and unfortunately the bot is still unable to function.
